topics/sequence-analysis/tutorials/quality-contamination-control/tutorial.md

```diff
@@ -25,18 +26,29 @@ tags:
 - microgalaxy
 level: Introductory
 edam_ontology:
-- topic_3673 # Whole genome sequencing
-- topic_0622 # Genomics
-- topic_3301 # Microbiology
-- topic_3697 # Microbial ecology
```

Comment:
The comments were automatically removed by our yaml parser, should probably restore them